sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1227,8 +1227,18 @@ Task<Response<BlobProperties>> GetPropertiesAsync(bool async, CancellationToken
             // Download the entire stream
             async Task<Response<BlobDownloadInfo>> DownloadStreamAsync(bool async, CancellationToken ct)
             {
-                Response<BlobDownloadInfo> response = await client.DownloadAsync(accessConditions: blobAccessConditions, cancellationToken: ct).ConfigureAwait(false);
-                await response.Value.Content.CopyToAsync(destination, 81920 /* default value */, ct).ConfigureAwait(false);
+                Response<BlobDownloadInfo> response = await client.DownloadInternal(
+                    default, blobAccessConditions, default, async, cancellationToken).ConfigureAwait(false);
+
+                if (async)
+                {
+                    await response.Value.Content.CopyToAsync(
+                        destination, 81920 /* default value */, ct).ConfigureAwait(false);
```

Comment:
Consider moving 81920 to Azure.Storage.Common.Constants.

Reply:
I only moved it. I do not know what it means and so don't know what to call it. @tg-msft?

Reply:
Yeah...  this was a fun one.  Different versions of the .NET framework use different default buffer sizes for `CopyToAsync` which caused a couple of tiny deltas in recorded tests.  We worked around it by fixing the default buffer size to the value in .NET Core.  It's fine to make this a constant if we want.